components/brave_shields/core/browser/ad_block_filters_provider_manager.cc

```diff
@@ -89,7 +89,7 @@ void AdBlockFiltersProviderManager::LoadFilterSetForEngine(
     task_tracker_.PostTask(
         base::SequencedTaskRunner::GetCurrentDefault().get(), FROM_HERE,
         base::BindOnce(&AdBlockFiltersProvider::LoadFilterSet,
-                       provider->AsWeakPtr(), std::move(collect_and_merge)));
```

Comment:
I'm not sure what is happening about this in production, but it looks like this was wrong, as we do `std::move(collect_and_merge)` for the first provider, but we keep looping for the next ones, and after the first we have a moved-from callback.